src/back-end/lib/permissions.ts

```diff
@@ -227,8 +227,21 @@ export function publishSWUOpportunity(session: Session): boolean {
   return isAdmin(session);
 }
 
-export async function deleteSWUOpportunity(connection: Connection, session: Session, opportunityId: string): Promise<boolean> {
-  return isAdmin(session) || (session && isGovernment(session) && await isSWUOpportunityAuthor(connection, session.user, opportunityId)) || false;
+export async function deleteSWUOpportunity(connection: Connection, session: Session, opportunityId: string, status: SWUOpportunityStatus): Promise<boolean> {
+  return isAdmin(session) && [SWUOpportunityStatus.Draft, SWUOpportunityStatus.UnderReview].includes(status)
+    || (session && isGovernment(session) && await isSWUOpportunityAuthor(connection, session.user, opportunityId) && status === SWUOpportunityStatus.Draft) || false;
```

Comment:
You don't need the `session &&` bit

Reply:
It throws a type error on the following call if I don't due to TypeScript weirdness.  I've changed it to use isSignedIn() for consistency.